frameworks/g-cloud-12/questions/services/virtualisationTechnologiesUsed.yml

```diff
@@ -30,10 +30,10 @@ options:
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Select what virtualisation technologies are used.
   - name: under_10_words
-    message: You can’t write more than 10 words for each virtualisation technology.
+    message: Each virtualisation technology must be 10 words or fewer.
   - name: max_items_limit
     message: You must have 10 or fewer virtualisation technology.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?